core/renderers/common/drawer.ts

```diff
@@ -321,7 +328,13 @@ export class Drawer {
     if (this.info_.isInsertionMarker) {
       // Fields and icons are invisible on insertion marker.  They still have to
       // be rendered so that the block can be sized correctly.
-      svgGroup.setAttribute('display', 'none');
+      if (isIcon((fieldInfo as Icon).icon)) {
```

Comment:
😵‍💫

Reply:
Haha I'll do another look at cleaning this up, but I think this method just needs to be split into some submethods

Reply:
I'm assuming that when you remove handling the old types some of these if statements will go away (though maybe not here since you didn't leave the comment about removing it?) so I tried not to comment on that aspect of them, but this one got me lol. I don't think you really need to change anything here unless you think it's actually possible to improve htis. but don't bother if it will be removed in 7042